styles/widgets/common/scheduler.less

```diff
@@ -25,6 +25,12 @@
     .dx-scrollable-content {
         padding-top: @SCHEDULER_POPUP_SCROLLABLE_CONTENT_PADDING;
     }
+
+    .dx-form.dx-recurrence-editor-container {
```

Comment:
Can we collect the all recurrence editor CSS in the `recurrenceEditor.less` file?

Reply:
Yes, we already have such files. And for theme-based css too! But this code is in .dx-scheduler-appointment-popup parent section.